<?xml version="1.0" ?>
<launch>
  <node name="schunk_pg70" pkg="schunk_pg70" type="schunk_pg70" output="screen" >
    <rosparam>
      gripper_id: 12
      portname: /dev/ttyUSB0
      baudrate: 9600  
    </rosparam> 
	</node>

	<node name="pg70_control_position" pkg="schunk_pg70" type="pg70_control_position" output="screen">
    <param name="current" value="10.0"/>
	</node>


  
  <group ns="schunk_pg70" >
  <node name="Grasp" pkg="schunk_pg70" type="pg70_action_server" output="screen" >
   <rosparam>
      gripper_id: 12
      portname: /dev/ttyUSB0
      baudrate: 9600  
   </rosparam> 

   <param name="current" value="10.0"/>
	</node>
  

</group>




</launch>